packages/core/useWebWorker/index.ts

```diff
@@ -22,38 +22,41 @@ export function useWebWorker(
   } = options
 
   const data: Ref<any> = ref(null)
-  let worker: Worker
+  let _worker: Worker
+  const worker = shallowRef<Worker>()
 
-  const post: typeof worker.postMessage = function post(val: any) {
-    if (!worker)
+  const post: typeof _worker.postMessage = function post(val: any) {
```

Comment:
```suggestion
  const post: typeof Worker['postMessage'] = function post(val: any) {
```

Would this work?

Reply:
Doesn't seem to, gives the following error

```
Property 'postMessage' does not exist on type '{ new (stringUrl: string | URL, options?: WorkerOptions | undefined): Worker; prototype: Worker; }'
```